intel-sgx/fortanix-sgx-tools/src/bin/ftxsgx-runner.rs

```diff
@@ -47,6 +48,30 @@ fn catch_sigbus() {
     }
 }
 
+#[cfg(unix)]
+fn catch_sigusr1() {
+    unsafe {
+        extern "C" fn handle_sigusr1(_signo: c_int, _info: *mut siginfo_t, _context: *mut c_void) {
+            eprintln!("SIGUSR1 triggered, thread_id: {:?}", std::thread::current().id());
```

Comment:
I wonder, should we use `eprintln!`, or should we add `log` and use logging? E.g. to make sure that a single configuration for logging output catches different parts in the same format nicely.

It's used in the fortanix-vme/fortanix-vme-runner module, for example, I think.

Leaving it to the author's discretion, no hard opinions.

Reply:
I don't think this needs to be logged. In many cases it doesn't even do anything.

Reply:
atm there are similar functions with eprintln, so i use it here also